third_party/blink/renderer/modules/webaudio/audio_node_output.cc

```diff
@@ -41,6 +41,7 @@ AudioNodeOutput::AudioNodeOutput(AudioHandler* handler,
       number_of_channels_(number_of_channels),
       desired_number_of_channels_(number_of_channels) {
   DCHECK_LE(number_of_channels, BaseAudioContext::MaxNumberOfChannels());
+  recordreplay::RegisterPointer("AudioNodeOutput", this);
```

Comment:
Can we have a comment explaining why we're registering this?  Also, there should be a destructor for the class that unregisters the pointer.